`graphblas/alloc.hpp: No such file or directory` on compile example

When trying to follow the instructions under _very quick start_:
```
cd <ALP/root/dir>
mkdir build
cd build
../bootstrap.sh --prefix=../install
make -j
make -j install
source ../install/bin/setenv
grbcxx ../examples/sp.cpp
grbrun ./a.out
```
The following error appears:
```
In file included from /tmp/ALP/install/include/graphblas/vector.hpp:32,
                 from /tmp/ALP/install/include/graphblas.hpp:527,
                 from ../examples/sp.cpp:36:
[...]/include/graphblas/reference/vector.hpp:50:10: fatal error: graphblas/alloc.hpp: No such file or directory
   50 | #include <graphblas/alloc.hpp>

```
Probably caused by #383

--- a/include/CMakeLists.txt
+++ b/include/CMakeLists.txt
@@ -41,7 +41,7 @@ set( HEADERS_REGEX ".+\.(hpp|h|hxx|hh|h\\+\\+)$" )
 # to avoid flaky acrobatics with regex or glob expressions, copy main files directly
 install( FILES "graphblas.hpp" DESTINATION "${INCLUDE_INSTALL_DIR}" )
 set( root_files
-	"graphblas/backends.hpp" "graphblas/benchmark.hpp"
+	"graphblas/alloc.hpp" "graphblas/backends.hpp" "graphblas/benchmark.hpp"
 	"graphblas/blas0.hpp" "graphblas/blas1.hpp" "graphblas/blas2.hpp"
 	"graphblas/blas3.hpp" "graphblas/collectives.hpp" "graphblas/config.hpp"
 	"graphblas/coordinates.hpp" "graphblas/descriptors.hpp"

